third_party/move/move-model/src/builder/module_builder.rs

```diff
@@ -1353,11 +1354,11 @@ impl<'env, 'translator> ModuleBuilder<'env, 'translator> {
                     et.set_translate_move_fun()
                 }
                 let loc = et.to_loc(&body.loc);
```

Comment:
Move this closer to its one use down a few lines?

Reply:
This seems to be just outside the 3-line loop that uses it.